deploy/setup.py

```diff
@@ -11,21 +11,8 @@
     license="Apache License 2.0",
     classifiers=["License :: OSI Approved :: Apache Software License"],
     packages=find_packages(exclude=["*.tests", "*.tests.*", "tests.*", "tests"]),
-    include_package_data=True,
     python_requires=">=3.6",
     install_requires=[
-        "docker[ssh]>=3.7.3",
-        "google-api-python-client>=1.12.1",
-        "paramiko>=2.4.2",  # explicitly pull in paramiko to prevent DistributionNotFound error
-        "docker-compose>=1.13.0",
-        "determined-common==0.14.6.dev0",
-        # requests<2.22.0 requires urllib3<1.25, which is incompatible with boto3>=1.14.11
-        "requests>=2.22.0",
-        # botocore>1.19.0 has stricter urllib3 requirements than boto3, and pip will not reliably
-        # resolve it until the --use-feature=2020-resolver behavior in pip 20.3, so we list it here.
-        "urllib3>=1.25.4,<1.26",
-        "termcolor==1.1.0",
-        "tqdm",
+        "determined==0.14.6.dev0",
     ],
-    entry_points={"console_scripts": ["det-deploy = determined_deploy.__main__:main"]},
```

Comment:
I also think that the `det-deploy` executable belongs to the `determined_deploy` package, and that the main repo should only contain `det deploy`

Reply:
I was thinking of `det deploy` as a separate chunk of work, for a later, separate PR. Then keeping `det-deploy` in `determined_deploy` defeats the whole purpose of merging it into the main package.
We can keep both `det-deploy` and `det deploy` in the main package to ease the migration, print a deprecation in `det-deploy`, and then eventually remove it.